test/credit_check_test.rb

```diff
@@ -6,26 +6,65 @@
 class CreditCheckTest < Minitest::Test
 
   def test_it_exists
-    cc = CreditCheck.new
+    cc = CreditCheck.new("5541808923795240")
 
     assert_instance_of CreditCheck, cc
   end
 
+  def test_digits
+    cc = CreditCheck.new("5541808923795240")
+
+    assert_equal [0, 4, 2, 5, 9, 7, 3, 2, 9, 8, 0, 8, 1, 4, 5, 5], cc.digits
```

Comment:
for readability, I usually like to reverse the use of variables from what you have.  I mean, you can spell out the actions more like this:
```
exptected = [0, 4, 2, 5, 9, 7, 3, 2, 9, 8, 0, 8, 1, 4, 5, 5]
actual = CreditCheck.new("5541808923795240").digits
assert_equal expected, actual
```
or even
```
exptected = [0, 4, 2, 5, 9, 7, 3, 2, 9, 8, 0, 8, 1, 4, 5, 5]
assert_equal expected, CreditCheck.new("5541808923795240").digits
```
I guess the main thing is that I see `CreditCheck.new("5541808923795240").digits` all together